index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Subrat Dash!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">    
+</head>
+<body>
+    <header>
+        <h2>Welcome</h2>

```

Comment:
- [ ] Hi, You've done some incredible work on your toolbar 👍. Although as part of the requirements you should have the Logo text that is in your `h2` use an `a` tag. If you look at most sites, when you click on the Logo on any page it will take you to the homepage. This type of functionality will really make your site that much better.
Something like this would be really nice.
```suggestion
        <a href="/"><h2>Welcome</h2></a>
```